lib/mbta.js

```diff
@@ -4,7 +4,30 @@
 
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
 
-};
+  let Subway = {
```

Comment:
Nice this is a great way to structure the data.  I would have used const here though because this object will not be changing